example/memcached-operator/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/operatorkit/example/memcached-operator/memcached"
+)
+
+func main() {
+	err := mainWithError()
+	if err != nil {
+		panic(fmt.Sprintf("%#v\n", err))
+	}
+}
+
+func mainWithError() (err error) {
+	c := parseFlags()
+
+	// Create a new logger that is used by all packages.
+	var newLogger micrologger.Logger
+	{
+		c := micrologger.Config{
+			IOWriter: os.Stdout,
+		}
+		newLogger, err = micrologger.New(c)
+		if err != nil {
+			return microerror.Maskf(err, "micrologger.New")
+		}
+	}
```

Comment:
I'm using micrologger here because it's used in OperatorKit. I know we want to have a text logger rather than JSON. Should we make that configurable in micrologger?